index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse</h1>
+  </body>
+</html>
```

Comment:
### Your Linters are not passing all the tests
![linters-not-passing](https://user-images.githubusercontent.com/93039359/185210521-8348c5e5-dd67-40c7-9610-ad2c8412cd15.png)

- [ ] You linters are not passing all the tests. Please remove the linter errors to make them test pass. :+1: Please follow the linters because linters prevent you from producing those errors which can break your application :fearful: